src/graphics.rs

```diff
@@ -1375,7 +1380,7 @@ impl SpriteBatch {
         let bindings = mq::Bindings {
             vertex_buffers: vec![ctx.quad_bindings.vertex_buffers[0], instances],
             index_buffer: ctx.quad_bindings.index_buffer,
-            images: vec![**texture.load_cached()],
+            images: vec![(**texture.load_cached()).texture],
```

Comment:
should be no need for explicit deref here, `load_cached` returns `&T` and `.texture` will continue to deref until it hits the right type or a type error